.github/workflows/publish-page.yml

```diff
@@ -24,10 +24,15 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      
+      - name: Install pnpm
```

Comment:
Out of curiousity: Why pnpm and not npm or yarn?

Reply:
In general i find pnpm faster than both yarn and npm. 

one of the things i like the most about pnpm however is how it manages `node_modules`. somewhat similarly to how `yarn pnp`, instead of writing files to it, it creates hard links to its internal store where all modules live. This means no huge `node_modules` directories, but also that you are able to install new dependencies also offline if you happen to have them in the store by doing `pnpm i --offline`. While i understand this a bit of a niche use case i found it very useful in a few occasions (especially when you start to have a few modules in the store).

The biggest one however is how dependencies are (or not) hoisted in node_modules. with pnpm, your module cannot access modules that are not explicitly set as dependencies, which eliminates a whole range of silly bugs when using modules not listed in your package.json. your dependencies however are not affected and can still access anything inside node_modules (this is to maximize compatibility with dependencies you have no control over). see https://pnpm.io/npmrc#shamefully-hoist.

it may not be extremely beneficial in this repo which is fairly simple, but i simply tend to default to it when i can. Happy to go with yarn or npm however if we you prefer

Reply:
Thanks :) Yes, these sound like great features!